src/modules/flow/freegeoip/freegeoip.json

```diff
@@ -14,7 +14,7 @@
         "open": "freegeoip_open",
         "close": "freegeoip_close"
       },
-      "name": "location/freegeoip",
+      "name": "freegeoip/location",
```

Comment:
ok, so we should change the filename to location.c, otherwise the .so will be freegeoip and the runner will no find the .so

I still prefer this, we can put the other modules that implement the same interface in the same category. I don't we are going to add other service under the freegeoip.json/c